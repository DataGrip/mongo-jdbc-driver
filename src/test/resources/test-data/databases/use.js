// command
use "my-db";
// command
db.coll.find();
// clear
use admin