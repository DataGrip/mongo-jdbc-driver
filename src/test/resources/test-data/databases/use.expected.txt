|result|
|switched to db my-db|

|map|